src/goto-symex/symex_goto.cpp

```diff
@@ -48,8 +48,8 @@ void goto_symext::symex_goto(statet &state)
     !instruction.targets.empty(), "goto should have at least one target");
 
   // we only do deterministic gotos for now
-  if(instruction.targets.size()!=1)
-    throw "no support for non-deterministic gotos";
+  DATA_INVARIANT(
+  	instruction.targets.size()!=1, "no support for non-deterministic gotos");
```

Comment:
Again, this hasn't been inverted properly, should be `instruction.targets.size() == 1`